packages/nvidia_nat_atif/src/nat/atif/scripts/export_trajectory_to_phoenix/README.md

```
@@ -0,0 +1,128 @@
+| Flag | Default | Description |
+|------|---------|-------------|
+| `files` (positional) | *(required)* | One or more ATIF trajectory JSON files |
+| `--endpoint` | `http://localhost:6006/v1/traces` | Phoenix server endpoint URL |
+| `--project` | `atif-trajectories` | Phoenix project name for trace grouping |
+| `--verbose`, `-v` | off | Enable debug logging |
+
+## Architecture
+
+### Processing flow
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)